src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -247,6 +247,8 @@ interfaces:
       parameters:
       - shelf.name=Math
       - series_uuid.series_string=xyz3141592654
+      sample_args:
```

Comment:
We haven't decided on this format. Let's sit down with the design doc and decide on one so we minimize backtracking.

Reply:
Yeah I did the easiest thing to make progress. Sitting down sounds good.